templates/registration/profile.html

```diff
@@ -23,11 +23,16 @@ <h2 class="profile-header">Welcome, {% firstof user.get_full_name user.get_usern
             </div>
 
             {% if total_earned %}
-                <p class="profile-total">Total earned: ${{ total_earned|floatformat:2|intcomma }}</p>
+                <h5 class="profile-total-header">TOTAL EARNED:</h5>
+                <h5 class="profile-total"> ${{ total_earned|floatformat:2|intcomma }}</h5>
             {% else %}
-                <p class="profile-total">Your money is going towards making some sweet music!</p>
+                <h5 class="profile-total-header">TOTAL EARNED:</h5>
+                <h5 class="profile-total">Your money is going toward creating some amazing music!
+                    <br/>It usually takes a few weeks from the time the music is finished to when revenue is reported...
```

Comment:
I wouldn't use an ellipsis here. It makes the statement come across as less trustworthy.

Also, just a thought but this seems to be a lot of text for a `<h5>`. Are you sure something like this should just be a normal `<p>`?